### DISCUSSION TITLE
isValidating for Specific Fields

### DISCUSSION THREAD

**Discussion by Alntjan** [react-hook-form/react-hook-form#6583] [Ideas]
So, I've been trying to a form with loading states for my async validations. But I can only know if the whole form is validating and not just the fields that are being validated still, for example, the async validations take more time to validate so the input in those situations should have a loading state.
It would be awesome to have a `isValidating` object like the `dirtyFields`, something like `validatingFields` that would have all the fields still being validated 😄 

PS: If there is another way of doing this I'm all 👂 👂

**Comment by bluebill1049** [comment]
here is a relevant feature request: https://github.com/react-hook-form/react-hook-form/issues/5744 (which you could upvote) personally I think it's easier to implement such a state at your app (field) level.
